MSALiOSB2C/ViewController.swift

```diff
@@ -66,7 +66,9 @@ class ViewController: UIViewController, UITextFieldDelegate, URLSessionDelegate
              The scheme part, i.e. "msal<your-client-id-here>", needs to be registered in the info.plist of the project
              */
             
-            let pcaConfig = MSALPublicClientApplicationConfig(clientId: kClientID)
+            let authority = try self.getAuthority(forPolicy: self.kSignupOrSigninPolicy)
+            
+            let pcaConfig = MSALPublicClientApplicationConfig(clientId: kClientID, redirectUri: nil, authority: authority)
```

Comment:
nit: do we want to specify redirectUri or put in a comment what nil means here?

Reply:
addressed